sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -365,6 +374,16 @@ export interface CreateAndRunThreadOptions {
     truncationStrategy?: TruncationObject | null;
 }
 
+// @public (undocumented)
```

Comment:
What this means is that this type is missing doc-comments and should be added ideally (same goes for all properties in this type)👍